.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "matrix-doc"]
```

Comment:
submodules are not fetched automatically you need to ask git to do that, so if you haven't pull matrix-doc it just runs tests without providing you with spec-coverage. 

On the other-hand if we move towards using the matrix-doc to also create / respond to mock requests (we don't currently) than indeed you would need matrix-doc to run tests.